README.md

```diff
@@ -23,33 +23,45 @@ bundle exec rake spec
 ## Usage
 ```
 Usage:
-  imagemaster3000 start --binaries-guestfish=BINARIES-GUESTFISH --binaries-virt-copy-in=BINARIES-VIRT-COPY-IN --certificate=CERTIFICATE --definitions-repository=DEFINITIONS-REPOSITORY --endpoint=ENDPOINT --group=GROUP --image-dir=IMAGE-DIR --image-list=IMAGE-LIST --key=KEY
+  imagemaster3000 start --binaries-guestfish=BINARIES-GUESTFISH --binaries-virt-copy-in=BINARIES-VIRT-COPY-IN --certificate=CERTIFICATE --definitions-repository=DEFINITIONS-REPOSITORY --endpoint=ENDPOINT --group=GROUP --image-dir=IMAGE-DIR --image-list-path=IMAGE-LIST --key=KEY
 
 Options:
-  --definitions-repository=DEFINITIONS-REPOSITORY  # Repository from which image definitions will be downloaded
-  [--definitions-branch=DEFINITIONS-BRANCH]        # Repository branch that will be used
-  --image-dir=IMAGE-DIR                            # Directory where to temporarily store images
-                                                   # Default: /var/spool/imagemaster3000/images/
-  --group=GROUP                                    # Group, images will be uploaded to
-                                                   # Default: imagemaster3000
-  --image-list=IMAGE-LIST                          # Name and path of generated image list
-                                                   # Default: /var/spool/imagemaster3000/image-list/imagemaster3000.list
-  --endpoint=ENDPOINT                              # Endpoint where image list will be available
-                                                   # Default: http://localhost/
-  --certificate=CERTIFICATE                        # Certificate to sign image list with
-                                                   # Default: /etc/grid-security/cert.pem
-  --key=KEY                                        # Key to sign image list with
-                                                   # Default: /etc/grid-security/key.pem
-  --binaries-virt-copy-in=BINARIES-VIRT-COPY-IN    # Path to binary needed for 'copy' action
-                                                   # Default: /usr/bin/virt-copy-in
-  --binaries-guestfish=BINARIES-GUESTFISH          # Path to binary needed for 'remove' action
-                                                   # Default: /usr/bin/guestfish
-  --logging-level=LOGGING-LEVEL
-                                                   # Default: ERROR
-                                                   # Possible values: DEBUG, INFO, WARN, ERROR, FATAL, UNKNOWN
-  [--logging-file=LOGGING-FILE]                    # File to write logs to
-                                                   # Default: /var/log/imagemaster3000/imagemaster3000.log
-  [--debug], [--no-debug]                          # Runs in debug mode
+--definitions-repository=DEFINITIONS-REPOSITORY  # Repository from which image definitions will be downloaded
```

Comment:
When you are generating help you have to temporarily remove your configuration because `Thor` will use it to provide defaults in the help output, e.g. your repository, your branch and so on. Please fix that.